fe/fe-core/src/main/java/org/apache/doris/datasource/hive/source/HiveScanNode.java

```diff
@@ -413,6 +413,12 @@ public TFileFormatType getFileFormatType() throws UserException {
             if (serDeLib.equals(HiveMetaStoreClientHelper.HIVE_JSON_SERDE)
                     || serDeLib.equals(HiveMetaStoreClientHelper.LEGACY_HIVE_JSON_SERDE)) {
                 type = TFileFormatType.FORMAT_JSON;
+            } else if (serDeLib.equals(HiveMetaStoreClientHelper.OPENX_JSON_SERDE)) {
+                if (hmsTable.canReadHiveJsonInOneColumn()) {
```

Comment:
I think we should return error if `READ_HIVE_JSON_IN_ONE_COLUMN` is true but the first column is not string?